src/frontend/src/lib/canisters/signer.canister.ts

```diff
@@ -42,7 +45,9 @@ export class SignerCanister extends Canister<SignerService> {
 			certified: true
 		});
 
-		const response = await btc_caller_address({ network, address_type: { P2WPKH: null } }, []);
+		const response = await btc_caller_address({ network, address_type: { P2WPKH: null } }, [
```

Comment:
Should we also extract a constant for `{ P2WPKH: null }`?

Reply:
Possibly.  @lmuntaner , what do you think?